### PR TITLE
Give zIndex to Tooltip component

### PR DESCRIPTION
**Pull request by sanderlooijenga**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206193208017747

### CODE REVIEW

File: packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const base = style([
       borderBottom: `6px solid ${tokens.kda.foundation.color.background['layer-3'].default}`,
       borderLeft: '6px solid transparent',
     },
+    zIndex: tokens.kda.foundation.zIndex.overlay,
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts:27]: do you think we should add the zIndex tokens to atoms?

Comment (sanderlooijenga) [reply packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts:27]: I mean, yeah we could! Right now it has `[-1, 0, 1]` as options. If you think it's helpful, I will implement it in the atoms 👍

Comment (eileenmguo) [reply packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts:27]: I don't have a strong preference, so it's up to you 👍

**Review by eileenmguo — APPROVED** [review]

File: packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts (hunk 1 above)
Comment (salamaashoush) [reply packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts:27]: I think the values in the tokens are not generic and are specific to some components hence they should not be a utility class

Comment (sanderlooijenga) [reply packages/libs/react-ui/src/components/Tooltip/Tooltip.css.ts:27]: That's true, so leave them out of the atoms (at least for now)?

**Pull request merged**